README.md

```diff
@@ -3,60 +3,70 @@
 Galasa provides Helm charts to install various components, the main one being a Galasa Ecosystem.
 
 ## Usage
+**Note: The Galasa Ecosystem chart only supports x86-64 at the moment. It cannot be installed on ARM64-based systems.**
 
 [Helm](https://helm.sh) must be installed to use the charts.  Please refer to
 Helm's [documentation](https://helm.sh/docs) to get started.
 
 Once Helm has been set up correctly, add the repo as follows:
 
-  helm repo add galasa https://galasa-dev.github.io/helm
+```
+helm repo add galasa https://galasa-dev.github.io/helm
+```
 
 If you had already added this repo earlier, run `helm repo update` to retrieve
-the latest versions of the packages.  You can then run `helm search repo galasa` to see the charts.
+the latest versions of the packages. You can then run `helm search repo galasa` to see the charts.
 
+If you would like to install the chart into minikube, ensure you have minikube [installed](https://minikube.sigs.k8s.io/docs/start/) and that it is running with `minikube status`. If minikube is not running, start it by running `minikube start`. 
 
 ## Galasa Ecosystem chart
+### RBAC
+If RBAC is active on your Kubernetes cluster, you will need to get your Kubernetes administrator to modify and apply an RBAC yaml file [(rbac-admin.yaml)](./charts/ecosystem/rbac-admin.yaml) to allow authorised users to run the helm install/upgrade/delete commands to interact with the helm chart.
```

Comment:
How should they modify it ? 
What permissions do they need to assign ? to which userid ?

Reply:
Addressed in https://github.com/galasa-dev/helm/pull/1/commits/fad3b94c4581df0dae003f50aac19569f88aa2f1